csm_web/frontend/static/frontend/css/style.css

```diff
@@ -72,6 +73,14 @@ header {
 	color: #bbbbbb;
 }
 
+#site-subtitle-link {
```

Comment:
Make this a class. Unless there is specific functionality that requires this to be an id, a class should be used.

Reply:
Done! Also copied some of the code from site-title-link into site-subtitle-link and removed site-title-link references from policies navlink